nservicebus/messaging/headers.md

```diff
@@ -34,30 +34,50 @@ public static DateTime ToUtcDateTime(string wireFormattedString)
 }
 ```
 
-
 ## Serialization Headers
 
 This set of headers contains information to control how messages are [de-serialized](/nservicebus/serialization/) by the receiving endpoint.
 
- * `NServiceBus.ContentType`: The type of serialization used for the message. For example ` text/xml` or `text/json`. The `NServiceBus.ContentType` header was added in Version 4.0. In some cases it may be useful to use the `NServiceBus.Version` header to determine when to use the `NServiceBus.ContentType` header.
- * `NServiceBus.EnclosedMessageTypes`: The fully qualified type name of the enclosed message(s).
+### NServiceBus.ContentType
+The type of serialization used for the message. For example ` text/xml` or `text/json`. The `NServiceBus.ContentType` header was added in Version 4.0. In some cases it may be useful to use the `NServiceBus.Version` header to determine when to use the `NServiceBus.ContentType` header.
+
+### NServiceBus.EnclosedMessageTypes
+The fully qualified .NET type name of the enclosed message(s). The receiving endpoint will use this type when deserializing an incoming message. Depending on the [versioning strategy](/samples/versioning/) the type can be specified in the following ways:
+
+* Full type name: `Namespace.ClassName`
+* Assembly qualified name: `Namespace.ClassName, AssemblyName, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null`
 
+NOTE: In integration scenarios, this header can be safely omitted if the endpoint uses XmlSerialization and the root node happens to message type.
```

Comment:
I was trying to incorporate @andreasohlund's comment here, however I am not sure how to make it sound right for the Json serialization. @Particular/docs-maintainers - feel free to fix up the note. 

>Also should we mention that you can leave out the header completely if
You are using XML and the root node == messagetype
json.net and $type=messagetype ?